src/main/java/utils/command/AddCardToDeckCommand.java

```diff
@@ -38,6 +39,8 @@ private void addCardToDeck(DeckList deckList, CardList cardList, UserInterface u
             ui.printDeckCreationSuccess();
             deckToAdd = new Deck(deckName, cardUUID);
             deckList.addDeck(deckToAdd);
+        } else if(deckToAdd.cardInDeck(cardUUID) == true) {
```

Comment:
can just write `else if (deckToAdd.cardInDeck(cardUUID))`

Reply:
Issues resolved in edab2bc2b42c131e9ab63358f987ffae5538ebed